src/tests/signup.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import SignUp from '../views/sign-up/SignUp';
+
+describe('Signup Page', () => {
+	test('renders the signup form', () => {
+		render(<SignUp />);
+		expect(screen.getByLabelText('signup-form')).toBeInTheDocument();
```

Comment:
provide more test assertions to make sure that the component functionality is working well.